src/lemlib/motions/turnTo.cpp

```diff
@@ -101,8 +102,12 @@ void turnTo(std::variant<Angle, V2Position> target, Time timeout, TurnToParams p
                         helper.getDelta());
 
         // move the motors
-        settings.leftMotors.move(-motorPower);
-        settings.rightMotors.move(motorPower);
+        if (!params.lockedSide || *params.lockedSide == TurnToParams::LockedSide::RIGHT) {
+            settings.leftMotors.move(-motorPower);
+        }
+        if (!params.lockedSide || *params.lockedSide == TurnToParams::LockedSide::LEFT) {
+            settings.rightMotors.move(motorPower);
+        }
```

Comment:
The motors should be commanded to brake every iteration of the loop if they are locked. Also the conditional logic can be made to be more expressive

```suggestion
        settings.leftMotors.move(-motorPower);
        settings.rightMotors.move(motorPower);
        // check which side of the drivetrain to lock, if any
        if (params.lockedSide) {
            if (*params.lockedSide == TurnToParams::LockedSide::LEFT) {
                settings.leftMotors.brake();
            } else {
                settings.rightMotors.brake();
            }
        }
```